streams-contrib/streams-persist-elasticsearch/src/main/java/org/apache/streams/elasticsearch/ElasticsearchPersistWriter.java

```diff
@@ -321,6 +316,27 @@ public void add(String indexName, String type, String id, String ts, String json
         add(indexRequestBuilder.request());
     }
 
+    /**
```

Comment:
Is this function necessary? If so we need to fix it, otherwise it should be removed.